src/web/layouts/CommentLayout.tsx

```diff
@@ -249,6 +249,7 @@ export const CommentLayout = ({ CAPI, NAV }: Props) => {
                         currentNavLink={NAV.currentNavLink}
                         pillar={CAPI.pillar}
                     />
+                    <GuardianLines pillar={CAPI.pillar} />
```

Comment:
We can drop these out to their own section - they're not really part of SubNav at all and then we have the bonus that they don't need to be rehydrated on the client